fossunited/chapters/doctype/foss_chapter_event/foss_chapter_event.py

```
@@ -591,4 +591,48 @@ def get_context(self, context):
+
+    import requests
+
+    if not map_url:
+        return None, None
+
+    # Follow redirects for shortened URLs (goo.gl, maps.app.goo.gl)
+    if "goo.gl" in map_url:
+        try:
+            response = requests.head(map_url, allow_redirects=True, timeout=5)
+            map_url = response.url
+        except Exception:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)